### PR TITLE
Reinstate Checkout Pack-shots

### PR DESCRIPTION
**Pull request by Ap0c**
# Why?

The pack-shots for the subs checkouts were hidden as part of Garnett. We now have new assets, so we can show them again. The changes are currently on CODE if anyone wants to have a look.

# Changes

- Started showing images on the checkouts again.
- Added new Garnett versions of the checkout images.
- Optimised the digital image.

# Screenshots

*DigiPack*

https://subscribe.code.dev-theguardian.com/checkout?countryGroup=uk

checkout-digipack

*Paper*

https://subscribe.code.dev-theguardian.com/checkout/voucher-sixday

checkout-paper

*Weekly*

https://subscribe.code.dev-theguardian.com/checkout/weeklyzonea-guardianweeklyquarterly?countryGroup=us

checkout-weekly

### CODE REVIEW

File: app/views/support/PlanOps.scala [hunk 1]
```diff
@@ -46,17 +46,17 @@ object PlanOps {
...
       case Digipack :: Nil =>
         ResponsiveImageGroup(
-          availableImages = Seq(ResponsiveImage(controllers.CachedAssets.hashedPathFor("images/digital-pack.png"), 300)),
+          availableImages = Seq(ResponsiveImage(controllers.CachedAssets.hashedPathFor("images/digital-pack-garnett.png"), 300)),
             altText = Some("Guardian apps demoed on Apple and Android devices")
         )
       case Weekly :: Nil =>
         ResponsiveImageGroup(
-          availableImages = ResponsiveImageGenerator("f3f9792bbb2c2f04ef01d0c1022e04eb87a2619a/0_0_4350_3416", Seq(500, 1000)),
+          availableImages = ResponsiveImageGenerator("987daf55251faf1637f92bffa8aa1eeec8de72b5/0_0_1670_1558", Seq(500, 1000), "png"),
```
Comment (paulbrown1982) [inline app/views/support/PlanOps.scala]: Wouldn't the photo images be smaller as JPEGs?

File: app/views/support/PlanOps.scala [hunk 2]
```diff
@@ -46,17 +46,17 @@ object PlanOps {
...
           altText = Some("Stack of The Guardian Weekly editions")
         )
       case _ =>
         ResponsiveImageGroup(
-          availableImages = ResponsiveImageGenerator("05129395fe0461071f176f526d7a4ae2b1d9b9bf/0_0_5863_5116", Seq(140, 500, 1000, 2000)),
+          availableImages = ResponsiveImageGenerator("67db3892d52a59ce6916942a7d2cb0828e5ffc15/0_0_2724_2154", Seq(140, 500, 1000, 2000), "png"),
```
Comment (paulbrown1982) [inline app/views/support/PlanOps.scala]: Wouldn't the photo images be smaller as JPEG?

**Comment by paulbrown1982** [comment]
Given that the crop sizes are small for the packshots (500px typically @ ~ 90KB), there would be little to gain by changing to JPEG. If we were using crops > 1000 I'd be more concerned.

👍 Thanks for reinstating these.

**Comment by Ap0c** [comment]
@paulbrown1982 We decided to drop the Observer from the paper image to make it consistent with what was shown previously (i.e. a Guardian paper only). This is now a jpg 🙂. I also optimised the digital image a bit, it was higher res than it probably needed to be.

**Pull request merged**